tests/cross_fw/test_templates/test_bias_correction.py

```diff
@@ -143,9 +144,14 @@ def quantized_test_model(self, tmpdir) -> TModel:
             (ConvTestModel, {"/conv/Conv": [0.11085186, 1.0017344]}),
             (DepthwiseConvTestModel, {"/conv/Conv": [-1.1229, -0.1863]}),
             (TransposeConvTestModel, {"/conv/ConvTranspose": [0.66797173, -0.7070703]}),
+            (OneDimMM, {"/linear/MatMul": [0.95773065, 1.3218939, 0.81694865]}),
         ),
     )
     def test_update_bias(self, model_cls, ref_biases, tmpdir):
+        from nncf.quantization.algorithms.bias_correction.onnx_backend import ONNXBiasCorrectionAlgoBackend
+
+        if self.get_backend() is ONNXBiasCorrectionAlgoBackend and model_cls is OneDimMM:
+            pytest.skip("ONNX does not support BC with MM ops")
```

Comment:
Please check backend specific parts not in template, there is no guaranty that backend can be imported here